### ISSUE TITLE
NoMethodError 'reason'

### ISSUE THREAD

**Issue by owahlen** [codekitchen/logstash-input-kinesis#5]
We are using _logstash-input-kinesis_ within a docker container (in Amazon ECS) built with the following Dockerfile:

```
FROM logstash:1.5.4-1
RUN plugin install --version 1.4.0 logstash-input-kinesis
RUN plugin install --version 0.1.0 logstash-output-amazon_es
```

Unfortunately in our logs we see the following exception nearly every second:

```
Oct 11, 2015 3:58:01 PM com.amazonaws.services.kinesis.clientlibrary.lib.worker.ShutdownTask call
SEVERE: Application exception. 
org.jruby.exceptions.RaiseException: (NoMethodError) undefined method `reason' for #<Java::ComAmazonawsServicesKinesisClientlibraryTypes::ShutdownInput:0x220cf90b>
    at LogStash::Inputs::Kinesis::Worker.shutdown(/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-input-kinesis-1.4.0-java/lib/logstash/inputs/kinesis/worker.rb:34)
    at LogStash$$Inputs$$Kinesis$$Worker_1186006010.shutdown(LogStash$$Inputs$$Kinesis$$Worker_1186006010.gen:13)
```

Is there a version incompatibility or what could be the reason for this issue?

**Comment by codekitchen** [comment]
Hm, this looks like a bug in a PR that I just merged in, it's using the wrong method name https://github.com/awslabs/amazon-kinesis-client/blob/master/src/main/java/com/amazonaws/services/kinesis/clientlibrary/types/ShutdownInput.java#L41

I'll push up a fix

**Comment by codekitchen** [comment]
OK the fix has been released as version 1.4.1

https://github.com/codekitchen/logstash-input-kinesis/commit/055172ab92af9401f235a57d7cea77b391853da2

**Comment by owahlen** [comment]
This was fast.
Thank you!
